src/pages/trade/ui/trades/my-trades.tsx

```diff
@@ -0,0 +1,17 @@
+import { observer } from 'mobx-react-lite';
+import { connectionStore } from '@/shared/model/connection';
+import { useLatestSwaps } from '../../api/latest-swaps';
+import { TradesTable } from '@/pages/trade/ui/trades/table';
+import { NotConnected } from './not-connected';
+
+export const MyTrades = observer(() => {
+  const { subaccount, connected } = connectionStore;
+  // TODO: filter out irrelevant swaps from pindexer
+  const { data, isLoading, error } = useLatestSwaps(subaccount);
```

Comment:
todo: pindexer might return 2+ swaps per asset pair and a block height. Only one swap must be filtered out to match the user's swap. Might need some help do to 2+ swaps in the same block